CONTRIBUTING.md

```diff
@@ -54,16 +54,18 @@ contributions via GitHub pull requests. When contributing code please do the
 following:
 
 * Fork the repository and make your changes in a feature branch.
+* Please include new or update existing PlantUML sequence/architecture diagrams
+  that relate to your PR. See `./docs/diagrams/src/README.md` for details on how
+  we maintain our diagrams)
 * Please include unit and integration tests for any new features and updates to
-existing tests.
+  existing tests.
 * Please ensure the unit and integration tests run successfully. Refer to the
-Testing section in the [README.md](README.md) on how to run these
-tests.
-* Please ensure that lint passes by running `./bin/run_lint`. The command should
-produce no output if there are no lint errors. The linter requires all Python
-dependencies be installed locally (if not it'll throw import-errors). Install
-dependencies locally by running `pip3 install -r requirements.txt` from the
-root project directory.
+  Testing section in `/README.md` on how to run these tests.
+* Please ensure that lint passes by running `./bin/run_lint`. The linter requires
+  all Python dependencies be installed locally in a virtual environment, as detailed
+  in the [developer setup](https://github.com/hyperledger/sawtooth-next-directory/wiki/Developer-Setup)
```

Comment:
You referenced the wiki here instead of the Sphinx Docs. You'll want this: https://sawtooth-next-directory.readthedocs.io/en/latest/developer-setup.html